Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -42,6 +43,7 @@ import Color from "../../Core/Color.js";
  * @param {Number} [options.colorBlendAmount=0.5] Value used to determine the color strength when the <code>colorBlendMode</code> is <code>MIX</code>. A value of 0.0 results in the model's rendered color while a value of 1.0 results in a solid color, with any value in-between resulting in a mix of the two.
  * @param {Number} [options.featureIdAttributeIndex=0] The index of the feature ID attribute to use for picking features per-instance or per-primitive.
  * @param {Number} [options.featureIdTextureIndex=0] The index of the feature ID texture to use for picking features per-primitive.
+ * @param {Object} [options.pointCloudShading] Options for constructing a {@link PointCloudShading} object to control point attenuation based on geometric error and lighting.
```

Comment:
As you point out here, I was going for symmetry with `Cesium3DTileset` since that would indeed be a breaking change otherwise. Not sure which way is best here...

Though another thing to point out is I'm used to seeing it accessed through the tileset after construction, e.g. `tileset.pointCloudShading.attenuation = true` without using the constructor.

I'm leaning towards passing in just the options, as at least this is consistent.